src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
```

Comment:
```suggestion
      <fieldset class="fieldset">
        <legend>Personal information</legend>
```
Don't add blank line between child and parend elements but add blank line between multiple sibling blocks